app/views/components/_email-signup-section.html.erb

```diff
@@ -0,0 +1,45 @@
+<%
+  details ||= {}
+  link_text ||= false
+  link_href ||= false
+  intro ||= false
+  full_width ||= false
+  track_action ||= false
+  track_label ||= false
+  transparent_background ||= false
+  css_classes ||= false
+  classes ||= []
+  tracking_data = track_action ? {
+    module: "track-click",
+    "track-category": "emailAlertLinkClicked",
+    "track-action": track_action,
+    "track-label": track_label
+  } : {}
+
+  classes << css_classes if css_classes
+  classes << "app-c-email-signup--full-width" if full_width
+  classes << "app-c-email-signup--no-intro" if !intro
```

Comment:
This can be also written as `classes << "app-c-email-signup--no-intro" unless intro` which I think is more readable.